signer-service/src/api/services/pendulum.service.js

```diff
@@ -108,14 +126,26 @@ exports.sendStatusWithPk = async () => {
       if (remainingMaxSubsidiesAvailable.lt(SUBSIDY_MINIMUM_RATIO_FUND_UNITS)) {
         isTokensSufficient = false;
         console.log(`Token ${token} balance is insufficient.`);
+
+        slackNotifier.sendMessage({
+          text: `Current balance of funding account is ${nativeToDecimal(
+            remainingMaxSubsidiesAvailable,
+          ).toString()} ${token} please charge this account ${fundingAccountKeypair.address}.`,
+        });
       }
     }),
   );
 
   const minimumBalanceFundingAccount = multiplyByPowerOfTen(Big(PENDULUM_FUNDING_AMOUNT_UNITS), apiData.decimals);
   const nativeBalance = Big(balance?.free?.toString() ?? '0');
+
   if (nativeBalance.gte(minimumBalanceFundingAccount) && isTokensSufficient) {
     return { status: true, public: fundingAccountKeypair.address };
   }
+  slackNotifier.sendMessage({
```

Comment:
It could be that we reach this point just because `isTokensSufficient` is false – in that case we already sent an error message above.

We only need to send this message if `nativeBalance.gte(minimumBalanceFundingAccount)`.

Furthermore, don't forget the second argument `apiData.decimals` for `divideByPowerOfTen`.